specification/batch/Azure.Batch/routes.tsp

```diff
@@ -80,7 +80,7 @@ interface Pools {
         start time of the last aggregation interval currently available.
         """)
       @query
-      @clientName("starttime", "!autorest")
+      @clientName("starttime", "csharp,python,javascript,go")
```

Comment:
the change looks good to me, just curious why not in client.tsp?

Reply:
This be inside an unnamed model. Now it is possible to rename property in alias, but it still require some refactor to the file. I guess I would not want to do that for this small fix.

Reply:
maybe `!java`?

Reply:
Sorry, already merged.

Checked with Wei, both actually be valid.